app/presenters/filters_presenter.rb

```diff
@@ -20,10 +20,33 @@ def summary_items
   end
 
   def reset_url
-    "#"
+    return nil unless any_filters?
+
+    finder_url_builder.url_except(all_filter_params)
   end
 
 private
 
   attr_reader :facets, :finder_url_builder
+
+  def applied_filters
+    @applied_filters ||= facets.flat_map(&:applied_filters)
+  end
+
+  # Returns a hash of all relevant query parameters for the currently active filters from all
+  # facets, so they can be removed in one go.
+  def all_filter_params
+    applied_filters
+      .map { _1[:query_params] }
```

Comment:
```suggestion
      .map { |filter| filter[:query_params] }
```

I feel like the _1 syntax is sufficiently uncommon that this could be a "wtf" for the team that owns it and given Ruby 3.4 is adding a more idiomatic one it doesn't necessarily seem like syntax that will inevitably become mainstream.